src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodosContext } from '../../contexts/TodoProvider';
+
+type Props = {
```

Comment:
Use interface prefer.
Type has unique ability but if you don`t need them use interface. It is sampled in scaled and has more interesting features.